cmds/core/msr/doc.go

```diff
@@ -0,0 +1,83 @@
+// Copyright 2018-2020 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// msr -- read and write MSRs with regular command or Forth
+//
+// Synopsis:
+//     msr [OPTIONS] r glob MSR
+//     msr [OPTIONS] w glob MSR value
+//     msr [OPTIONS] forth-word [forth-word ...]
+//
+// Description:
+//    This program reads and writes sets of MSRs, while allowing
+//    them to by changed on a core by core or collective basis.
+//
+//    To read the msrs for 0 (sorry, the command is msr and the forth command
+//    is msr, making this a bit confusing):
+//    sudo msr 0 msr 0x3a reg rd
+//    Breaking that down:
+//    0 - for cpu 0
+//    msr - for take the glob, in this case 0, and push all matching filenames on the stack
+//    0x3a - for register 0x3a
+//    reg - convert to 32-bit integer and push
+//    rd - pop a 32-bit int and a []string and use them to read 1 or more MSRs
+//
+//    for all:
+//    sudo msr "'*" msr 0x3a reg rd
```

Comment:
That should be `"'"*`, right?

Reply:
I just tested and either one works ... I think I'll leave this as is, it's actually less confusing.